### ISSUE TITLE
Error on variable classes

### ISSUE THREAD

**Issue by rmand97** [100phlecs/tailwind_formatter#24]
The following code:
```
<tr>
  <th>ETA/ETD</th>
  <td>
    <%= for {vessel, eta, etd} <- list do %>
      <p class={"qa-thingy-etas-#{vessel.id} has-margin-bottom-sm"}>
        <span class={"qa-thingy-etas-#{vessel.id}-vessel"}><%= vessel.name %></span>
        (<span class={"qa-thingy-etas-#{vessel.id}-eta"}><%= convert_date?(
            eta,
            "Not Specified"
          ) %></span> &ndash; <span class={"qa-thingy-etas-#{vessel.id}-etd"}><%= convert_date?(
            etd,
            "Not Specified"
          ) %></span>)
      </p>
    <% end %>
  </td>
</tr>
<tr>
  <th>Deliverables</th>
  <td>
    <%= for {fuel, vessel_fuels} <- list,
              vessel_fuel <- vessel_fuels do %>
      <p class={"qa-thingy-deliverable-#{vessel_fuel.id} has-margin-bottom-sm"}>
        <span class="qa-thingy-deliverable-quantity"><%= vessel_fuel.quantity %></span>
        MT of <span class="qa-thingy-deliverable-fuel"><%= fuel.name %></span>
        to
        <span class="qa-thingy-deliverable-vessel">
          <%= vessel_fuel.vessel.name %> (<%= vessel_fuel.vessel.imo %>)
        </span>
      </p>
    <% end %>
  </td>
</tr>

```

gives me this error:

```
** (ArgumentError) errors were found at the given arguments:

  * 1st argument: not a textual representation of an integer

    :erlang.binary_to_integer("0$-vessel")
    lib/tailwind_formatter.ex:146: TailwindFormatter.get_sort_position/1
    lib/tailwind_formatter.ex:157: anonymous fn/1 in TailwindFormatter.sort_base_classes/1
    (elixir 1.14.2) lib/enum.ex:1658: Enum."-map/2-lists^map/1-0-"/2
    lib/tailwind_formatter.ex:156: TailwindFormatter.sort_base_classes/1
    lib/tailwind_formatter.ex:124: TailwindFormatter.sort/1
    lib/tailwind_formatter.ex:51: anonymous fn/1 in TailwindFormatter.format/2
    (elixir 1.14.2) lib/regex.ex:756: Regex.apply_list/5
```

**Comment by 100phlecs** [comment]
Thanks for the report.

This was due to the new `$` terminator on placeholders.

I just published a new release [(v0.3.4)](https://github.com/100phlecs/tailwind_formatter/releases/tag/v0.3.4) with your code as a test case. 

Could you try that and see if any other errors crop up?

**Comment by rmand97** [comment]
Seems to work now. I'm currently investigating using this library in production, so i might come back with more :)
